linguatec_lexicon_frontend/static/js/base.js

```diff
@@ -0,0 +1,51 @@
+$(function () {
+
+    init_dictionary_button();
+
+    function init_dictionary_button(){
+        var selected_dictionary = $("#selected_lex").val();
+        if (selected_dictionary == 'es-ar'){
+            $(".button-dictionary-change").removeClass('ar-es');
+            $(".button-dictionary-change").addClass('es-ar');
+            $(".button-dictionary-change").data("dictionary", "to-ar-es");
+            $('#input-search').attr('placeholder', 'castellano-aragonés');
+        } else {
+            $(".button-dictionary-change").removeClass('es-ar');
+            $(".button-dictionary-change").addClass('ar-es');
+            $(".button-dictionary-change").data("dictionary", "to-es-ar");
+            $('#input-search').attr('placeholder', 'aragonés-castellano');
+        }
+    }
+
+    $(".button-dictionary-change").click(function() {
+        var selected_dictionary = $(this).data("dictionary").replace("to-","");
+        $('#selected_lex option[value="' + selected_dictionary + '"]').prop('selected', true);
+        init_dictionary_button();
+    });
```

Comment:
IMHO it's simpler to store the current status of the button and not the next status. I mean, if the button shows background "es-ar" data will be to "es-ar".

This way we could remove the prefix "to-".

Besides that, I think that we could replace the lexicon input selector by a hidden input (maybe embedded on button-dictionary-change).

And, a final comment about naming, "lexicon" is prefered than "dictionary".